### DISCUSSION TITLE
What to add to xaml view to be able to find Style StyletConductorTabControl?

### DISCUSSION THREAD

**Discussion by HackMasterFlash** [canton7/Stylet#388] [Q&A] [answered]
I have a TabControl that works as expected when I use the code snippet in the documentation. For completeness here is xaml ...

```
<UserControl x:Class="FODS_Config.Views.TabControlView"
            xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
             xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
             xmlns:s="https://github.com/canton7/Stylet"
             xmlns:vms="clr-namespace:FODS_Config.ViewModels"
             mc:Ignorable="d" 
             d:DataContext="{d:DesignInstance vms:TabControlViewModel}"
            d:DesignHeight="460" d:DesignWidth="714">
    <Grid>
        <TabControl ItemsSource="{Binding Items}" SelectedItem="{Binding ActiveItem}" DisplayMemberPath="DisplayName">
            <TabControl.ContentTemplate>
                <DataTemplate>
                    <ContentControl s:View.Model="{Binding}" VerticalContentAlignment="Stretch" HorizontalContentAlignment="Stretch" IsTabStop="False"/>
                </DataTemplate>
            </TabControl.ContentTemplate>
        </TabControl>        
    </Grid>
</UserControl>
```
``
But when I try to use the text saving style shortcut ...

`<TabControl Style="{StaticResource StyletConductorTabControl}" />`

VisualStudio tells me 
    The resource "StyletConductorTabControl" could not be resovled. 

The QuickStart tells me:
It's worth noting that <s:ApplicationLoader> above is a ResourceDictionary subclass. This allows it to load in Stylet's built-in resources (see [Screens and Conductors](https://github.com/canton7/Stylet/wiki/Screens-and-Conductors)).

I just don't know how to tell Visual studio where to find the StaticResource. Thanks for any help!

**Comment by canton7** [comment, marked as answer]
Does it actually run fine if you start the application? I.e. is it just a VS issue?

**Reply by HackMasterFlash** [reply]
Note to self. Don't freak out if VS puts a squiggly underline under something. Compile and run. Then freak out.
As it turned out VS's left hand didn't know about its right hand, in other words I should have just tried it because yes in fact it runs just fine. Thanks for the included Style!

And also in general, I appreciate a little less magic than Caliburn.Micro and like with this style some reduction of boiler plate! I'm feeling a little bit of pain trying to figure out what magic Caliburn did that I need to do implement explicitly as I convert an app to Stylet. So far not too bad. A lot of replacing a name which was automagically bound to a straight up binding. The trickier one was an alternate to the bubble up method handling magic. I'm going with just firing an event. Anyway thanks for a cool framework!

**Reply by canton7** [reply]
Great! In fairness, the style is in a ResourceDictionary which is in a separate DLL, and which is loaded in a slightly obscure way.
